sdks/python/README.md

```diff
@@ -78,13 +79,44 @@ client.get_chunks_by_url(
     url="https://example.com/document"
 )
 
-# Document Name Search : retrieve all chunks of a document identified by its name
+# Document Name Search: retrieve all chunks of a document identified by its name
```

Comment:
It'd be good to get some examples of the query filters for the methods that support them